WatchFaceAlphaKotlin/app/src/main/java/com/example/android/wearable/alpha/utils/ComplicationUtils.kt

```diff
@@ -80,20 +80,18 @@ fun createComplicationSlotManager(
     drawableId: Int = DEFAULT_COMPLICATION_STYLE_DRAWABLE_ID
 ): ComplicationSlotsManager {
 
-    val defaultComplicationDrawable = ComplicationDrawable.getDrawable(context, drawableId)
-
-    val canvasComplicationFactory =
+    val canvasLeftComplicationFactory =
```

Comment:
I think just removing the `defaultComplicationDrawable` should be enough here, and we can use the same `CanvasComplicationFactory` in both places.

Reply:
Moved it down to be separately called for each Factory and that works.